src/containers/RouteContainers/baseRouter.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright © 2019 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+import HomePage from '../HomePage';
+import ProductDetailPage from '../ProductDetailPage';
+import CategoryPage from '../CategoryPage';
+
+export default function b2cRoutes() {
```

Comment:
Can this be a constant instead of a function?

Reply:
altered